Assets/MRTK/Providers/LeapMotion/LeapMotionOrientationDisplay.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.LeapMotion.Input;
+using Microsoft.MixedReality.Toolkit.Input;
+using TMPro;
```

Comment:
The current location of this file is in the providers directory.  A better location would be in the example scene scripts directory here: 

https://github.com/microsoft/MixedRealityToolkit-Unity/tree/main/Assets/MRTK/Examples/Demos/HandTracking/Scripts